shuttle/config.yaml

```diff
@@ -1,18 +0,0 @@
-chains:
```

Comment:
why removed this file?

Reply:
Moved to specific upgrade folder since it'd be different for every upgrade testing